File: CheckCharacterCapitalOrNot.c
#include<stdio.h>
#include<stdbool.h>

bool CheckCapital(char ch)
{
    if((ch >= 'A') && (ch <= 'Z'))
    {
        return true;
    }
    else
    {
        return false;
    }
}

int main()
{
    char cValue = '\0';
    bool bRet = false;

    printf("Enter the character:\n");
    scanf("%c",&cValue);

    bRet = CheckCapital(cValue);
    if(bRet == true)
    {
        printf("%c is Capital case letter.\n",cValue);
    } 
    else
    {
        printf("%c is not capital case letter.\n",cValue);
    }

    return 0;
}

File: CheckCharacterCapitalOrNot.java
import java.util.*;

class CheckCapitalOrNot
{
    public char Ch;

    public CheckCapitalOrNot(char CValue)
    {
        Ch = CValue;
    }

    public boolean CheckCapital()
    {
        if((Ch >= 'A') && (Ch <= 'Z'))
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

class CheckCharacterCapitalOrNot
{
    public static void main(String arg[])
    {
        char cValue;
        boolean bRet = false;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the character:");
        cValue = sobj.nextLine().toCharArray()[0];

        CheckCapitalOrNot cobj = new CheckCapitalOrNot(cValue);

        bRet = cobj.CheckCapital();

        if(bRet == true)
        {
            System.out.println(cValue + " is Capital case letter.");
        }
        else
        {
            System.out.println(cValue + " is not capital case letter.");
        }
    }
}